SYCL/Basic/free_function_queries/free_function_queries.cpp

```diff
@@ -4,6 +4,9 @@
 // RUN: %CPU_RUN_PLACEHOLDER %t.out
 // RUN: %GPU_RUN_PLACEHOLDER %t.out
 
+// REQUIRED: TEMPORARY_DISABLED
```

Comment:
Why not UNSUPPORTED: windows?
I think you mentioned that it does pass on Linux. Right?

Reply:
BTW, I think we should file a bug report for missing compiler builds on Windows.

Reply:
Yes, but it is not guaranteed for all compilers.

Reply:
> Yes, but it is not guaranteed for all compilers.

Which compilers in addition to oneAPI DPC++ do you mean? 
I think from now on these tests should pass on Linux. Am I wrong?

Reply:
Take a look at pre-commit results for https://github.com/intel/llvm-test-suite/pull/667. Linux is clean and all these tests are failing on Windows only.